packages/bits/spec/components/repeat/repeat.visual.ts

```diff
@@ -0,0 +1,46 @@
+import { browser, by, ElementFinder } from "protractor";
+import { Atom } from "../../atom";
+import { Helpers } from "../../helpers";
+import { Camera } from "../../virtual-camera/Camera";
+import { RepeatAtom } from "./repeat.atom";
+
+const name: string = "Repeat";
+
+describe(`Visual tests: ${name}`, () => {
+    let camera: Camera;
+    let singleSelectList: RepeatAtom;
+    let singleSelectListRequired: RepeatAtom;
+    let reorderSelectList: RepeatAtom;
+    let toggleDragging: ElementFinder;
+
+    beforeAll(async () => {
+        await Helpers.prepareBrowser("repeat/repeat-visual-test");
+        singleSelectList = Atom.find(RepeatAtom, "nui-demo-single-highlight");
+        singleSelectListRequired = Atom.find(RepeatAtom, "nui-demo-single-required-selection");
+        reorderSelectList = Atom.find(RepeatAtom, "nui-demo-reorder-config");
+        toggleDragging = browser.element(by.css(".nui-switch__bar"));
+        camera = new Camera().loadFilm(browser, name);
+    });
+
+    it(`${name} visual test`, async () => {
+        await camera.turn.on();
+        await camera.say.cheese(`Default`);
+
+        await singleSelectList.hover();
+        await camera.say.cheese(`Repeat in Single Selection Mode with Radio Buttons and Required Single Selection Mode`);
+
+        await singleSelectListRequired.hover();
+        await camera.say.cheese(`Repeat in Single Selection Mode with Item Highlight`);
+
+        await reorderSelectList.hover();
+        await camera.say.cheese(`Repeat Item Customization`);
```

Comment:
Maybe this for better clarity?

```suggestion
        await camera.say.cheese(`Item Drag/Drop Enabled`);
```